webapp/src/components/tutorialValidators.tsx

```diff
@@ -53,21 +61,43 @@ export class BlocksExistValidator extends CodeValidatorBase {
             return defaultResult();
         }
 
+        // if no valid markers are specified, default to "highlight"
+        if (!this.markers || (!this.markers.includes("highlight") && !this.markers.includes("exists"))) {
+            this.markers = ["highlight"];
```

Comment:
I think I agree starting with `validate-exists` defaulting on as well is fine / just leaving this as an escape hatch if we don't want highlight.